text.go

```diff
@@ -528,25 +528,18 @@ func (t *Text) MostCommonFontFace() FontFace {
 	return family.Face(size*ptPerMm, col, style, variant)
 }
 
-// ToPaths makes a path out of the text, with x,y the top-left point of the rectangle that fits the text (ie. y is not the text base)
-func (t *Text) ToPaths() ([]*Path, []color.RGBA) {
-	paths := []*Path{}
-	colors := []color.RGBA{}
+// RenderLetters renders the text converted to paths (calling r.RenderPath)
+// Don't forget to call RenderDecoration as well
```

Comment:
The other option would be to call `t.RenderDecoration` at the end of this function (or is there any reason not to render the decoration when rendering the letters?)